tests/model_fields/test_integerfield.py

```diff
@@ -174,6 +174,15 @@ def test_rel_db_type(self):
         rel_db_type = field.rel_db_type(connection)
         self.assertEqual(rel_db_type, self.rel_db_type_class().db_type(connection))
 
+    def test_existence_of_min_and_max_validators(self):
+        """
+        Integer fields by default should have a MinValueValidtor
+        and a MaxValueValidator.
+        """
+        model_validators = self.model._meta.get_field("value").validators
+        self.assertIsInstance(model_validators[0], validators.MinValueValidator)
+        self.assertIsInstance(model_validators[1], validators.MaxValueValidator)
```

Comment:
This can fail for 3rd-party database backends. I don't there is any need for additional tests in this patch.